pkg/api/handlers/libpod/images_pull.go

```diff
@@ -182,6 +182,13 @@ loop: // break out of for/select infinite loop
 				if err := enc.Encode(report); err != nil {
 					logrus.Warnf("Failed to json encode error %q", err.Error())
 				}
+
+				report.Images = nil
+				report.ID = images[len(images)-1]
```

Comment:
I think we should only set it if `all == false` (or document which image it refers to).